scalastyle-config.xml

```diff
@@ -250,6 +250,14 @@ This file is divided into 3 sections:
     <customMessage>Omit braces in case clauses.</customMessage>
   </check>
 
+  <!-- SPARK-16877: Avoid Java annotations -->
+  <check customId="OverrideJavaCase" level="error" class="org.scalastyle.scalariform.TokenChecker" enabled="true">
+    <parameters><parameter name="regex">^Override$</parameter></parameters>
```

Comment:
I just reprored this locally on my machine as well (doesn't trigger with `@Override` or `\@Override`). One guess is the token checker has split them into separate tokens perhaps?